src/ui/pages/Identifiers/Identifiers.tsx

```diff
@@ -116,27 +115,33 @@ const Identifiers = () => {
   }, [currentOperation, history.location.pathname]);
   useEffect(() => {
     setShowPlaceholder(identifiersData.length === 0);
-    setAllIdentifiers(
-      identifiersData
-        .filter((identifier) => !identifier.isPending)
-        .filter((identifier) => !identifier.groupMetadata?.groupId)
-        .filter(
-          (identifier) =>
-            !favouritesIdentifiers?.some((fav) => fav.id === identifier.id)
-        )
-    );
-    setFavIdentifiers(
-      identifiersData.filter((identifier) =>
-        favouritesIdentifiers?.some((fav) => fav.id === identifier.id)
-      )
-    );
-    setPendingIdentifiers(
-      identifiersData.filter((identifier) => identifier.isPending)
-    );
-    setMultiSigIdentifiers(
-      identifiersData.filter((identifier) => identifier.groupMetadata?.groupId)
-    );
-  }, [favouritesIdentifiers, identifiersData, toggleClick]);
+    const tmpPendingIdentifiers = [];
+    const tmpMultisigIdentifiers = [];
+    const tmpFavIdentifiers = [];
+    const tmpAllIdentifiers = [];
+    for (const identifier of identifiersData) {
+      if (identifier.isPending) {
+        tmpPendingIdentifiers.push(identifier);
```

Comment:
I presume we only want to show it in one place @sdisalvo-crd - so I would put `continue` after each of these

Pretty sure things should never be in both places at once anyway, but not sure how the favourites plays with this...

Reply:
actually we can only favourite a fully created identifier i believe